pkg/sentry/fsimpl/fuse/BUILD

```diff
@@ -40,6 +40,7 @@ go_library(
         "register.go",
         "regular_file.go",
         "request_list.go",
+        "request_response.go",
```

Comment:
Do we still want to use this name when we now only have requests and responses for Init?

Reply:
@jinmouil do you need to place any non-abi structs in this new file? If not, I'll move the `fuseInitRes` to `init.go`.

Reply:
See https://github.com/google/gvisor/pull/3548

I have a lot to put. And `init.go` will become `connection_control.go`

Reply:
Cool thanks, then I'll keep this name.